tools/tcat_ble_client/ble/ble_stream_secure.py

```diff
@@ -102,6 +107,13 @@ async def do_handshake(self, timeout=30.0):
         else:
             print('TLS Connection timed out.')
             return False
+        print('')
```

Comment:
Is that print needed?

Reply:
Yes it is previously it looked like this in bbtc.py file:
```python
        try:
            await ble_sstream.do_handshake()
            print('\nDone')
            ble_sstream.log_cert_identities()
        except Exception as e:
            print('\nFailed')
            logger.error(e)
````
Plus it was not printing new line in case of scan command.

Reply:
makes sense :)